docs/Scheduler.md

```diff
@@ -0,0 +1,350 @@
+createdAt: Timestamp
+maxSurge: String | Integer
+portRange: PortRange
+forwarders: Forwarders
+spec: Spec
+```
+
+- **Name**: Scheduler name. This name is unique and will be the same name used for the kubernetes namespace.
+  It's offered by the user in the creation and cannot be changed in the future. It's used as an ID for the scheduler;
+- **game**: Name of the game which will use the scheduler. The game is important since it's common to use multiple schedulers
+  for a specific game. So you probably will want to fetch all the schedulers from a game;
+- **createdAt**: Info about the scheduler creation time. Cannot be altered by the user;
+- **maxSurge**: Value represented in percentage (%) or Integer. Offer by the user on the scheduler creation. Can be altered anytime.
+  Used by Maestro to replace pods. Ex: If maxSurge = 3, the [Switch Active Version](Operations.md#switch-active-version) (Major change) operation will replace pods from 3 to 3;
+- **portRange**: Range of ports that can be used by Maestro to create GRUs for the specified scheduler. Can be altered by the user anytime. More info [here](#portrange);
+- **forwarders**: Maestro can pass ahead info sent by the game rooms, such as Ping (Ready, Occupied, Terminating...), player and rooms events.
+  The receivers can be configured here. More info [here](#forwarders);
+- **spec**: Specifications about the game rooms managed by the scheduler, such as containers and environment variables used by them, limits and images. More info [here](#spec).
+
+
+### PortRange
+The **PortRange** is used to select a random port for a GRU between **start** and **end**.
+ 
+- PortRange cannot be null or empty
+
+> ⚠ Check if the ports offered are **available** and **can be used**. 
+> A firewall rule, for example, can affect the connection to the Game Room in the specific port.
+
+It is represented as:
+```yaml
+start: Integer
+end: Integer
+```
+
+### Forwarders
+Forwarders are configured to pass ahead info offered by the game rooms to Maestro. 
+More than one forwarder can be configured for a scheduler.
+
+- Can be an empty list.
+
+It is represented as:
+```yaml
+- name: String
+  enable: Bool
+  type: String
+  address: String
+  options:
+    timeout: Integer
+    metadata: Object
+```
+- **name**: Name of the forwarder. Used only for reference (visibility and recognition);
+- **enable**: Toggle to easily enable/disable the forwarder;
+- **type**: Type of the forwarder. Right now, only accepts **gRPC**;
+- **address**: Address used by the scheduler to forward events. E.g. 'api.example.com:8080';
+- **options**: Optional parameters.
+    - **timeout**: Timeout value for an event to successfully be forwarded;
+    - **metadata**: Object that can contain any useful information for the game team. Will be forwarded with the events from Maestro.
+
+### Spec
+Contains vital information about the game rooms. Be aware that the spec is the most related aspect of the scheduler interacting with kubernetes. 
```

Comment:
What do you think to change kubernetes here to `Runtime` or something like that?